sample/src/main/java/com/mvc/imagepicker/sample/MainFragment.java

```diff
@@ -53,22 +64,61 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View v = inflater.inflate(R.layout.fragment_main, container, false);
-        imageView = (ImageView) v.findViewById(R.id.image_view);
+        imageView1 = (ImageView) v.findViewById(R.id.image_view_1);
+        imageView2 = (ImageView) v.findViewById(R.id.image_view_2);
+        imageViewGalleryOnly = (ImageView) v.findViewById(R.id.image_view_gallery_only);
```

Comment:
I've changed the demo a little to show how to load different images in different views and use Picasso instead load plain BMP.